phpunit.xml.dist

```diff
@@ -23,7 +23,6 @@
             <directory suffix=".php">src/</directory>
             <exclude>
                 <file>src/LaravelServiceProvider.php</file>
```

Comment:
I think we don't need that anymore either.

Reply:
Ack, removed the whole `<exclude>` part